tools/eslint-rules/eslint-check.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @fileoverview Check that common.skipIfEslintMissing is used if
+ *               the eslint module is required.
+ */
+'use strict';
+
+const utils = require('./rules-utils.js');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+const msg = 'Please add a skipIfEslintMissing() call to allow this test to ' +
+            'be skippped when Node.js is built from a source tarball.';
```

Comment:
typo: skippped?

Reply:
I based this on the equivalent inspector-check and didn't notice it was wrong there. I'll correct both.